src/quo/components/wallet/network_routing/view.cljs

```diff
@@ -172,6 +173,16 @@
             [rn/view {:style (style/max-limit-bar-background network-name)}]
             [dashed-line network-name]])]))))
 
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map
+      [:networks {:optional true} [:maybe [:sequential :any]]]
```

Comment:
If we have all the networks in a collection, the schema could check if it is homogenous over it?

Reply:
Done